MapleServer2/Tools/PartyManager.cs

```diff
@@ -33,6 +33,11 @@ public Party GetPartyById(long id)
             return PartyList.TryGetValue(id, out Party foundParty) ? foundParty : null;
         }
 
+        public Party GetPartyByMember(long id)
```

Comment:
Almost every method in this class, with one function, can be done in Expression Body.

Reply:
teach me senpai

Reply:
if the method has a simple line of code, you can remove the `{ }` and instead at the end of the parameter you do `=>` that's a lambda expression, then you put  the return value.